### PR TITLE
Adds Page component to make our markup a little saner.

### PR DESCRIPTION
**Pull request by chuckharmston**
screen shot 2015-09-24 at 1 35 58 pm

No need for Windex. r? @ngokevin

### CODE REVIEW

File: src/media/css/page.styl [hunk 1]
```diff
@@ -1,12 +1,30 @@
-@import 'lib';
+@import('lib')
```
Comment (ngokevin) [inline src/media/css/page.styl]: meh

File: src/media/css/page.styl [hunk 2]
```diff
@@ -1,12 +1,30 @@
...
 
-.subnav
+
+.page
+  padding $page--spacing
+
+
+.page--header
+  align-items flex-end
+  border-bottom 1px solid #DDD
```
Comment (ngokevin) [inline src/media/css/page.styl]: use the palette

File: src/media/css/page.styl (hunk 1 above)
Comment (chuckharmston) [inline src/media/css/page.styl]: That wasn't intentionally, actually. I accidentally deleted it and added it back by hand.

**Comment by ngokevin** [comment]
r+ great work

**Pull request merged**